substrate/frame/support/procedural/Cargo.toml

```diff
@@ -38,18 +38,14 @@ codec = { features = [
 	"max-encoded-len",
 ], workspace = true }
 regex = { workspace = true }
-sp-metadata-ir = { workspace = true, features = ["std"] }
+sp-metadata-ir = { workspace = true }
 scale-info = { features = ["derive"], workspace = true }
-sp-core = { workspace = true, features = ["std"] }
-sp-io = { workspace = true, features = ["std"] }
-sp-runtime = { features = [
-	"runtime-benchmarks",
-	"serde",
-	"std",
-], workspace = true }
-frame-system = { workspace = true, features = ["runtime-benchmarks", "std"] }
-frame-support = { workspace = true, features = ["runtime-benchmarks", "std"] }
-frame-benchmarking = { workspace = true, features = ["std"] }
+sp-core = { workspace = true }
+sp-io = { workspace = true }
+sp-runtime = { features = ["runtime-benchmarks", "serde"], workspace = true }
+frame-system = { workspace = true, features = ["runtime-benchmarks"] }
+frame-support = { workspace = true, features = ["runtime-benchmarks"] }
```

Comment:
I think it should work without the `runtime-benchmarks` feature.

Reply:
It is because I use it in the doc

Reply:
I removed its usage, it leaked in the testing build and other crate couldn't compile